### ISSUE TITLE
IOR miscalculates bandwidth when time drift is large

### ISSUE THREAD

**Issue by richard-mansfield** [hpc/ior#431]
IOR miscalculates bandwidth if one or more compute nodes has a significant time drift.

IOR should either exit gracefully in this circumstance or use  Lamport timestamp to synchronized thread timings.

**Comment by JulianKunkel** [comment]
I do not believe that this is an IOR issue. Algorithmic-wise a detection of regular time drifts due to too fast/slow clocks is possible, however, there are unsteady ones too and I do not know any algorithm that can reliably deal with any type of clock errors - except if one *constantly* watches for time issues. However, any benchmark will have a similar challenge. Time synchronisation is something important for logs incl. security, so I believe there is no need to fix this unpleasantry but rather ensure that nodes have the right timing e.g. using NTP. Maybe there is a good reason not to time sync the nodes, though?
